### ISSUE TITLE
In `Media`, EXIF desciriptions should be displayed if they exists

### ISSUE THREAD

**Issue by bilogic** [nextcloud/android#13215]
### Is your feature request related to a problem? Please describe.
Media is not showing EXIF description even when it exists.

### Describe the solution you'd like
Media should show EXIF descriptions directly if it exists, this gives additional context that is useful to the viewer

### Describe alternatives you've considered
None

### Additional context
No

**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by joshtrichards** [comment]
We do display EXIF/metadata. Is the field you're looking for visible in the Web UI under *Details* within the *Photos* app?

**Comment by bilogic** [comment]
yes it is, but that is not *directly*. I mean, I think many would consider to be very helpful information, why hide it? Imagine an instagram/facebook photo with no captions.

**Comment by joshtrichards** [comment]
I get that. My query about the server-side piece was to make sure the specific field you're asking for is available server-side. The app gets the EXIF/metadata from the server.

**Comment by bilogic** [comment]
Ah got it. Thank you and looking forwarding to this feature :)
